tests/unit/RelationshipsPanel.spec.ts

```diff
@@ -0,0 +1,70 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
+import { RelationshipsPanel } from '@/components/relationships-panel'
+import { ApprovalType } from '@/components/approval-type'
+import VueRouter from 'vue-router'
+import { RelationshipTypes } from '@/enums/relationship-types'
```

Comment:
Don't use enums in unit tests. We want the tests to be fragile and thus to check enum values. That way, an error in the enum may be caught by a test -- otherwise the same error may be cancelled out and not noticed.

Reply:
Okay I removed it. Makes a lot of sense.